### DISCUSSION TITLE
Unable to resolve type SukiTheme from namespace clr-namespace:SukiUI;assembly=SukiUI

### DISCUSSION THREAD

**Discussion by weya3** [kikipoulet/SukiUI#278] [Q&A]
I am new to SukiUI,when the package installed, I am following https://kikipoulet.github.io/SukiUI/documentation/ this documentation but not able apply SukiUI theme. I have attached image. can someone check what have done wrong ? I have used SukiUI nuget package.
![image](https://github.com/user-attachments/assets/522ac937-da88-42e5-b939-c059bfb82030)
the version I had try any group include( avalonia 11.1.0 SukiUI 5.3)(avalonia 11.1.0-beta2 SukiUI 6.0.0-beta6 like document)(avalonia 11.1.2 SukiUI 6.0.0-beta6)(avalonia 11.1.3 SukiUI 6.0.0-beta7)by nuget
If the solution build  
![image](https://github.com/user-attachments/assets/591af817-6960-4f10-926c-2118bbd90f31)

**Comment by AuroraZiling** [comment]
try importing the SukiUI dll from the latest build of Github Action while using the latest Avalonia Beta version.

**Reply by weya3** [reply]
I uninstall the SukiUI package which installed by nuget;and imported by  SukiUI dll,updated  the latest Avalonia Beta version 11.2.0-beta1
now  there are the new falut
![image](https://github.com/user-attachments/assets/ab827153-e10b-4607-9a63-f02872368d86)
system can't find the file
I think maybe the project must need Reative UI?I build project by commuity toolkit
![image](https://github.com/user-attachments/assets/192d08b2-ac8b-4481-9a49-016fd8b20837)
there is project dependiencs
so import by this way， the project need install more dependiencs？

**Reply by AuroraZiling** [reply]
yes, dll from Github Actions needs System.Reactive

**Reply by weya3** [reply]
![image](https://github.com/user-attachments/assets/d40ddc58-325c-481e-9137-d80c32d3f4a9)
System.Reactive has been installed now,but the error message can't remove
![image](https://github.com/user-attachments/assets/cfc8c8cb-bccb-458b-a3dc-9b026a8f48f1)

App.axaml look like success
![Uploading image.png…]()

MainWindow has the same error message

**Reply by weya3** [reply]
咱们直接中文吧，才看见你是国人，哈哈哈，帮我瞅瞅是为啥这个

**Reply by weya3** [reply]
我清理项目并刷新了一下，重新build
![image](https://github.com/user-attachments/assets/e0f19900-7fbe-4bca-87f7-7b6e093a585c)
现在是这样子，无法preview，输出也有错误

**Comment by xlxdd** [comment]
我也碰到这个问题，最终还是放弃了

**Reply by kikipoulet** [reply]
Show us your nuget packages and the error

**Reply by xlxdd** [reply]
nuget packegs:
```xml
<ItemGroup>
        <PackageReference Include="Avalonia" Version="11.1.0"/>
        <PackageReference Include="Avalonia.Desktop" Version="11.1.0"/>
        <PackageReference Include="Avalonia.Themes.Fluent" Version="11.1.0"/>
        <PackageReference Include="Avalonia.Fonts.Inter" Version="11.1.0"/>
        <!--Condition below is needed to remove Avalonia.Diagnostics package from build output in Release configuration.-->
        <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.1.0"/>
        <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.1"/>
        <PackageReference Include="SukiUI" Version="5.3.0" />
        <PackageReference Include="System.Reactive" Version="6.0.0" />
</ItemGroup>
```
build error:
```
0>App.axaml(14,10): Error AVLN2000 Avalonia: Unable to resolve type SukiTheme from namespace clr-namespace:SukiUI;assembly=SukiUI Line 14, position 10.
```
and in avalonia 11.2.0 and SukiUI 6.0.1 failed again.

**Reply by kikipoulet** [reply]
Your SukiUI version is 5.3 ..
Just update all your nuget package to latest

**Reply by xlxdd** [reply]
Thank you,it works

**Reply by cymscyms** [reply]
I also encountered this problem, and the final research found that it was a 5.3 problem, and now the official documentation is in accordance with 6.0, so you need to update to 6.0, and you can search for 6.0.0 by checking "including pre-release version" in nuget management. After that, this problem does not arise anymore.
